CMakeLists.txt

```diff
@@ -402,6 +402,35 @@ if (NOT MINGW AND NOT APPLE)
     set (CMAKE_EXE_LINKER_FLAGS "-Wl,-rpath=${_qt5Core_install_prefix}/lib")
     # install desktop file
     install( FILES build/mscore.desktop DESTINATION share/applications)
+    # compress man pages if gzip is present on system
+    set(MAN_TARGET ${PROJECT_SOURCE_DIR}/build/mscore.1)
+    find_program( GZIP_EXECUTABLE gzip DOC "A tool for compressing manpages (optional)." )
```

Comment:
I don't think CMake has an "OpenBSD" variable so I am going to test this using [CMAKE_SYSTEM_NAME](http://www.cmake.org/cmake/help/v3.0/variable/CMAKE_SYSTEM_NAME.html) which stores a value equivalent to running `$ uname -s`.
Can you confirm for me that the output of running `$ uname -s` on OpenBSD is *exactly* "OpenBSD" and nothing else?

(Also, I read [here](http://openbsd.das.ufsc.br/faq/faq2.html#ManPages) that man pages can be compressed on OpenBSD, but I take it that's not standard practice?)